[
    {
        "content": "<p>I'm having trouble getting <code>createsequence.sql</code> to run: <a href=\"https://github.com/pdurbin/dataverse-api-test-runner/commit/52c655c\">https://github.com/pdurbin/dataverse-api-test-runner/commit/52c655c</a></p>",
        "id": 378472226,
        "sender_full_name": "Philip Durbin ðŸš€",
        "timestamp": 1690291119
    },
    {
        "content": "<p>I expected this to work:</p>\n<p><code>psql -h localhost -U $DATAVERSE_DB_USER $DATAVERSE_DB_NAME -f doc/sphinx-guides/source/_static/util/createsequence.sql</code></p>",
        "id": 378472359,
        "sender_full_name": "Philip Durbin ðŸš€",
        "timestamp": 1690291142
    },
    {
        "content": "<p>But I got this:</p>\n<p><code>psql: error: connection to server at \"localhost\" (::1), port 5432 failed: Connection refused</code></p>",
        "id": 378472524,
        "sender_full_name": "Philip Durbin ðŸš€",
        "timestamp": 1690291176
    },
    {
        "content": "<p>See <a href=\"https://github.com/pdurbin/dataverse-api-test-runner/actions/runs/5648925205/job/15302157873\">https://github.com/pdurbin/dataverse-api-test-runner/actions/runs/5648925205/job/15302157873</a></p>",
        "id": 378472648,
        "sender_full_name": "Philip Durbin ðŸš€",
        "timestamp": 1690291200
    },
    {
        "content": "<p><a href=\"https://github.com/pdurbin/dataverse-api-test-runner/commit/6cb1464\">https://github.com/pdurbin/dataverse-api-test-runner/commit/6cb1464</a> is getting me closer. Thanks, <span class=\"user-mention\" data-user-id=\"598442\">@Guillermo Portas</span> !</p>",
        "id": 378539571,
        "sender_full_name": "Philip Durbin ðŸš€",
        "timestamp": 1690302809
    },
    {
        "content": "<p>I needed this fix too: <a href=\"https://github.com/pdurbin/dataverse-api-test-runner/commit/85a986e\">https://github.com/pdurbin/dataverse-api-test-runner/commit/85a986e</a></p>",
        "id": 378543294,
        "sender_full_name": "Philip Durbin ðŸš€",
        "timestamp": 1690303605
    },
    {
        "content": "<p>All set. In short, ports instead of expose . And the env vars I was using were blank.</p>",
        "id": 378543543,
        "sender_full_name": "Philip Durbin ðŸš€",
        "timestamp": 1690303658
    },
    {
        "content": "<p>Bah. Spoke too soon. Next error is \"psql:doc/sphinx-guides/source/_static/util/createsequence.sql:20: ERROR:  role \"dvnapp\" does not exist\"</p>",
        "id": 378543907,
        "sender_full_name": "Philip Durbin ðŸš€",
        "timestamp": 1690303721
    },
    {
        "content": "<p>Ok, fixed: <a href=\"https://github.com/pdurbin/dataverse-api-test-runner/commit/7711ff4\">https://github.com/pdurbin/dataverse-api-test-runner/commit/7711ff4</a></p>",
        "id": 378551169,
        "sender_full_name": "Philip Durbin ðŸš€",
        "timestamp": 1690305169
    }
]